src/test/java/Homework16.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+@Test
+public class Homework16 extends BaseTest {
+    @Test
+    public static void registrationNavigation(){
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5
+        ));
+
+        String url= "https://bbb.testpro.io/";
+
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.cssSelector ("[id='hel']"));
+        registrationLink.click();
+
+        String urlRegistration = "https://bbb.testpro.io/Registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(),urlRegistration);
+        driver.quit();
+    }
+
+
+   }
+
```

Comment:
Please, add Homework16 in TestNG.xml  run tests

For example

![image](https://user-images.githubusercontent.com/104761482/221401525-47ac755d-4376-4217-87d4-cbb99860deb6.png)